test/Kestrel.Core.Tests/Http2ConnectionTests.cs

```diff
@@ -311,6 +313,10 @@ private void InitializeConnectionFields(MemoryPool<byte> memoryPool)
                 ServiceContext = new TestServiceContext()
                 {
                     Log = new TestKestrelTrace(_logger)
```

Comment:
I prefer using `new Mock<KestrelTrace>(Logger) { CallBase = true }` instead of adding a bunch of properties to TestKestrelTrace.